src/Microsoft.IdentityModel.Tokens/Validation/Results/Details/IssuerValidationError.cs

```diff
@@ -9,25 +9,36 @@ namespace Microsoft.IdentityModel.Tokens
 {
     internal class IssuerValidationError : ValidationError
     {
-        private string? _invalidIssuer;
-
         internal IssuerValidationError(
             MessageDetail messageDetail,
             Type exceptionType,
             StackFrame stackFrame,
             string? invalidIssuer)
-            : base(messageDetail, ValidationFailureType.IssuerValidationFailed, exceptionType, stackFrame)
+            : this(messageDetail, ValidationFailureType.IssuerValidationFailed, exceptionType, stackFrame, invalidIssuer, null)
         {
-            _invalidIssuer = invalidIssuer;
         }
 
+        internal IssuerValidationError(
```

Comment:
Multiple ctors with different numbers of optional parameters gets confusing fast.

This is internal, is it safe to change the above ctor? That is, because of the friend assemblies, is this already used in another assembly?

If it is not, can we just have the one ctor with both params optional?

See RS0026 as well

https://github.com/dotnet/roslyn-analyzers/blob/main/src/PublicApiAnalyzers/Microsoft.CodeAnalysis.PublicApiAnalyzers.md